src/css/components/Card.scss

```diff
@@ -0,0 +1,51 @@
+@import "../config/index.scss";
+
+//------------------------------------------------------------//
+
+// * Card Component
+// * 1. Define Default Card
```

Comment:
`Card` seems very similar in style to `Panel`. Do we need both?

Reply:
True, I'm just wondering if we'd like to make it it's own to keep formatting consistent across the app. If it's just panel, design details could vary a fair amount over time. These are the three I'm looking at right now. The third would be a modifier...

![cards](https://user-images.githubusercontent.com/1189536/56828250-350f1800-681e-11e9-8aa3-fe4b30551b79.jpg)

Initially I was thinking of Material's card design, but they don't have a panel. https://material.io/design/components/cards.html#anatomy

Reply:
Hrm... if there were some really specific font / layout styles just for cards I think it could make sense, but the styles added here don't quite  justify it IMO, since `.Card-content, -contentRow, and contentHeader` are all just adding minimal spacing. `Card-image` is the most compelling case for making it a component.

If you do make it it's own component, I'd suggest reducing the child classes that just add spacing to make it more versatile and easy to use.

Reply:
Fair points. I think if scaling back the card element to focus on the image, I might lean towards just doing an Image component or utilities. List too. I'll do that for now... It could be too that this level of component should be in React and not the CSS. My concern in general with scaling back to smaller components is that we're giving devs puzzle pieces that are less intuitive to put together.

Reply:
If you do stick with Panel, FWIW, I think `Panel-image` would be a useful child of Panel, though we may want variations on the max-height and how the image is positioned within the container (for example, center-aligned vs top-aligned) OR those attributes could be great candidates for adding on the React component based on context.

Reply:
Yeah it's hard to say. A Panel image wouldn't really work on large panels and I feel like a child element should work at all sizes. Kinda back and forth on it all too.